src/components/Sidebar/components/UserCard/UserCard.tsx

```diff
@@ -38,10 +38,10 @@ const UserCard = (props: Props) => {
       {show && (
         <ExpandedList>
           <ExpandedListItem>
-            <Link to="/settings">Profile & Settings</Link>
+            <StyledLink to="/settings">Profile & Settings</StyledLink>
```

Comment:
A cleaner way would be to wrap the styled text inside the Link
```suggestion
<Link to="/settings">
  <ComeUpWithAGoodName>
    Profile & Settings
  </ComeUpWithAGoodName>
</Link>
```
on a different note, it's not the best practice to call something StyledXYZ. Always better to call things after what they do or are, not what they are made off.

Reply:
Okey, I understand. Think I saw that it was a common thing to do in some other conversation. Or I just misunderstood everything

Reply:
so mby <NoDecorationLink> ?

Reply:
How about `<LinkText />` or something like that?
Cause it's not the link, just the text inside a link.

Reply:
That makes more sense! 😄

Reply:
@SimonNord @mo Can I just query this whole thing here for the craic! 

Could we not do this with a class? Maybe it's old school! I don't know. It just seems very specific and over the top to add a new component for this? Unless we can use this NoDecor Link elsewhere? Can we or will we or what's the theory? :/

Would it make any sense for <Link> to by default have no styles (i.e. take on the CSS from NoDecorLink) and then we add additional styles to it via class?